### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_CreateFunctionAppinAzure_ScreenReader: The NVDA is not announcing "Create Function App in Azure" header text of "Create Function App in Azure" dialog.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2942]
### Actual:
When we activate the "Create Function App in Azure" control the NVDA only announces "edit  Enter a globally unique name for the new function app. (Press 'Enter' to confirm or 'Escape' to cancel)  blank" but it does not announce the header text that is "Create Function App in Azure (1/3)".

**This issue is also observed with JAWS.
This issue is also observed with scenarios such as "Create new project" and "Create Function App in Azure(Advanced) (1/7)" dialog**

### Expected:
After activating the "Create Function App in Azure" control the NVDA should announce "Create Function App in Azure(1/3)" header text.

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev3;#WCAG1.3.1;#DesktopApp;#Win10;#NVDA;#JAWS;#Element:Header;#Linked:Bug138120;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1
Screen Reader: NVDA(2021.1),JAWS(2021.2107.12 ILM)

### Steps to Reproduce:
1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open. TAB to "Install" button and hit ENTER key to install it.
4. "Azure" button will appear in navigation pane. TAB to it and select it.
5. Start NVDA.
4. TAB to any Subscription and expand options using Shift+F10 then TAB to "Create Function App in Azure" control and select it.
5. Observe the announcement by screen reader after activating the "Create Function App in Azure" control. 

### User Impact:
The screen reader users won't be able to understand which popup or dialog they are navigating.

“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by nturinski** [comment]
This is a duplicate of https://github.com/microsoft/vscode-azurefunctions/issues/2691.  As mentioned in the original issue, please file an external issue to VS Code (https://github.com/microsoft/vscode/issues)

**Comment by 12-shweta** [comment]
@nturinski 
Logged a new external bug [https://github.com/microsoft/vscode/issues/138120](url) mentioned for above URL

**Comment by jinglouMSFT** [comment]
The VS Code bug was closed due to lack of a generic repro step.  The issue is similar to VS Code command: Git: Clone, where the content of the wizard is not read.

**Comment by 12-shweta** [comment]
@jinglouMSFT 
As you comment "The issue is similar to VS Code command: Git: Clone, where the content of the wizard is not read." so whether there is any bug logged to track this issue ?
And if this one is fixed then the primary bug will also be fixed?

**Comment by 12-shweta** [comment]
@jinglouMSFT 
Could you please look into the above comment

**Comment by 12-shweta** [comment]
@MicroFish91 The tracking issue has been closed but when we are checking on the Visual studio code we are still able to reproduce the issue, could you please provide build where the fixes will be available.

**Comment by MicroFish91** [comment]
Upstream issue we are tracking against is still open and being investigated, see previous comment's reference to issue 149387

**Comment by jinglouMSFT** [comment]
@MicroFish91 @12-shweta  VS Code has closed the bug as merged. Although the solution doesn't seem to be 100% reliable.  Can you try this in the Functions extension? Make sure you comment on #149387 in vscode.  Thanks.

**Comment by MicroFish91** [comment]
I ran and tested with 1.68 and the title field is now being picked up and read

**Comment by 12-shweta** [comment]
Verified the issue in 1.68 but still is reproducible.
![image](https://user-images.githubusercontent.com/54656883/172765449-67b987a9-f772-40ad-afe0-1eefd224486e.png)

**Comment by MicroFish91** [comment]
> Verified the issue in 1.68 but still is reproducible. ![image](https://user-images.githubusercontent.com/54656883/172765449-67b987a9-f772-40ad-afe0-1eefd224486e.png)

When I initially checked on a couple titles it was working, but I'm seeing that this use-case specifically is not working.  Let me identify which use-cases the aria label is not being read and discuss with the VS Code team.  I will get back to you after, thank you for your help!

**Comment by MicroFish91** [comment]
Short update, talked with Isidor in the posted issue thread above... solution will likely need to be resolved further upstream, so we will continue tracking, thanks

**Comment by MicroFish91** [comment]
Just a quick update, talked with Isidor and it looks like this will be investigated further in the November milestone for VS Code

**Comment by MicroFish91** [comment]
@12-shweta Hi, I was wondering if this could be downgraded to Sev3 based on new guidelines.  This issue has the same upstream tracking dependency as [this issue](https://github.com/microsoft/vscode-azurefunctions/issues/3123) which was also downgraded to Sev3.

Thank you!

**Comment by 12-shweta** [comment]
Downgrading the severity as per New Severity guidelines.

**Comment by jinglouMSFT** [comment]
@12-shweta This issue is showing up in S360 tracking in 12/5/2022's report. Can you make sure this issue has been correctly downgraded to Sev3?  Sev3 shouldn't be tracked in S360.  Thanks.

**Comment by 12-shweta** [comment]
We have lowered the severity of this bug but seems to be it got stale, will check with the team

**Comment by MicroFish91** [comment]
Upstream issue has been resolved, can you confirm and close if so?  Thank you!

**Comment by MicroFish91** [comment]
Closing as resolved by upstream issue, if you need to re-open for any reason please let me know, thank you
